model/ftn/w3adatmd.ftn

```diff
@@ -583,10 +584,10 @@
 !/PR3                                 MAPCXY(:), MAPTH2(:), MAPWN2(:)
 !/PR3      LOGICAL, POINTER        :: MAPTRN(:)
 !
-!/SMC      INTEGER, POINTER        :: NMX0, NMX1, NMX2, NMY0, NMY1, NMY2,  &
-!/SMC                                 NACT, NMXY
-!/SMC      INTEGER, POINTER        :: MAPX2(:), MAPY2(:), MAPAXY(:),       &
-!/SMC                                 MAPXY(:), MAPTH2(:), MAPWN2(:)
+!! !/SMC      INTEGER, POINTER        :: NMX0, NMX1, NMX2, NMY0, NMY1, NMY2,  &
```

Comment:
Do you think we need to still keep these lines commented out, or could we go ahead and delete these?

Reply:
I forgot to tidy them up after finishing the SMC multi-grid changes.  Yes, they are safe to be deleted.